README.md

```diff
@@ -13,7 +13,7 @@ As such, every user is free to use Aura and choose how to use it.
 Compatibility
 ------------------------------
 Compatibility to all major versions but NA was dropped on
-[2013-09-13 (Aura Legacy)](https://github.com/aura-project/aura_legacy/commit/c6483faace4d79b8f772bee519531718084a243d). Since that time, Aura is kept compatible only to the *latest update* of NA.
+[2013-09-13 (Aura Legacy)](https://github.com/saneki/aura_legacy/commit/c6483faace4d79b8f772bee519531718084a243d). Since that time, Aura is kept compatible only to the *latest update* of NA.
```

Comment:
My fork of legacy is more updated than the main if you want me to just link it to mine